Sources/Controllers/MessagesViewController.swift

```diff
@@ -60,6 +60,9 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
         return false
     }
 
+    /// The `BasicAudioController` controll the AVAudioPlayer state (play, pause, stop) and udpate audio cell UI accordingly.
+    open var audioController: BasicAudioController!
```

Comment:
Make this lazy, as those who don't use audio messages will never need it to be created
`open lazy var audioController = BasicAudioController(messageCollectionView: messageCollectionView)`